src/algebra/geom_sum.lean

```diff
@@ -172,6 +172,28 @@ theorem geom_sum₂_mul [comm_ring α] (x y : α) (n : ℕ) :
   (∑ i in range n, x ^ i * (y ^ (n - 1 - i))) * (x - y) = x ^ n - y ^ n :=
 (commute.all x y).geom_sum₂_mul n
 
+theorem sub_dvd_pow_sub_pow [comm_ring α] (x y : α) (n : ℕ) : x - y ∣ x^n - y^n :=
```

Comment:
```suggestion
theorem sub_dvd_pow_sub_pow [comm_ring α] (x y : α) (n : ℕ) : x - y ∣ x ^ n - y ^ n :=
```

Reply:
Fixed. One of the lines got pushed over 100 characters, so now `:=` is on its own line. (I think the Library Style Guidelines state that this is fine.)